info/v1/container.go

```diff
@@ -415,6 +415,9 @@ type FsStats struct {
 	// Number of bytes available for non-root user.
 	Available uint64 `json:"available"`
 
+	// HasInodes when true, indicates that Inodes info will be available.
+	HasInodes bool `json:"has_inodes"`
```

Comment:
So I have tried this change in my local workspace, but I am having trouble figuring out the best way to set it on `ContainerSpec` since best I can tell wherever we call `GetSpec` https://github.com/google/cadvisor/blob/master/container/common/helpers.go#L48, we don't yet appear to actually know if the associated filesystem supports inodes or not yet.